frontend/src/hooks/queries/studylog.ts

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios';
+import { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useMutation, useQuery } from 'react-query';
+import {
+  requestPostScrap,
+  requestDeleteScrap,
+  requestDeleteLike,
+  requestDeleteStudylog,
+  requestPostLike,
+} from '../../service/requests';
+import useSnackBar from '../useSnackBar';
+import {
+  ALERT_MESSAGE,
+  ERROR_MESSAGE,
+  SNACKBAR_MESSAGE,
+  SUCCESS_MESSAGE,
+} from '../../constants/message';
+import { PATH } from '../../constants';
+import { BASE_URL } from '../../configs/environment';
+import { UserContext } from '../../contexts/UserProvider';
+import { Studylog, StudyLogResponse } from '../../models/Studylogs';
+import { requestGetStudylogs } from '../../service/requests';
+
+const QUERY_KEY = {
+  recentStudylogs: 'recentStudylogs',
+  popularStudylogs: 'popularStudylogs',
+};
+
+export const useGetRecentStudylogsQuery = () => {
+  const { user } = useContext(UserContext);
+  const { accessToken } = user;
+
+  return useQuery<Studylog[]>([QUERY_KEY.recentStudylogs], async () => {
+    const { data } = await requestGetStudylogs({
+      query: { type: 'searchParams', data: 'size=3' },
+      accessToken,
+    }).then((res) => res.json());
```

Comment:
```suggestion
  return useQuery<Studylog[]>([QUERY_KEY.recentStudylogs], () => requestGetStudylogs({
      query: { type: 'searchParams', data: 'size=3' },
      accessToken,
    }).then((res) => res.json()).then(({data}) => data)
    
```
chaing과 await가 모두 사용되어 좀 어색한듯 한데, 한가지 방법으로 몰아줘도 좋을것같아요

Reply:
좋아요!
전체적으로 async/await 가 많이 쓰여서 이걸로 적용했습니다!

[refactor: then -> async/await로 수정](https://github.com/woowacourse/prolog/pull/977/commits/7e6abc16e3f2f6b3ebdaec6eecc3926410741892)